backend/src/main/java/com/team28/sidedish/utils/JwtUtils.java

```diff
@@ -0,0 +1,41 @@
+package com.team28.sidedish.utils;
+
+import com.auth0.jwt.JWT;
+import com.auth0.jwt.JWTVerifier;
+import com.auth0.jwt.algorithms.Algorithm;
+import com.auth0.jwt.interfaces.DecodedJWT;
+
+import java.util.Date;
+import java.util.Map;
+
+public class JwtUtils {
+
+    private static final Algorithm algorithm = Algorithm.HMAC256(EnvUtils.getJwtSecret());
+    private static final JWTVerifier verifier = JWT.require(algorithm).build();
+    private static final long expiredTime = 100 * 60 * 60 * 48L; // 48 시간
```

Comment:
유틸성 클래스가 아닌 JwtToken을 가진 클래스를 생성해서 생성, 검증 동작을 위임하는것도 좋은 방법일것 같아요.
```suggestion
public class JwtToken {
    private static final Algorithm algorithm = Algorithm.HMAC256(EnvUtils.getJwtSecret());
    private static final JWTVerifier verifier = JWT.require(algorithm).build();
    private static final long expiredTime = 100 * 60 * 60 * 48L; // 48 시간
    
    private final String token;

    public JwtToken(String email, String name){
        this.token = JWT.create()
        // ...
    }
        
    public boolean isValid(){
        // ...
    }    
        
```

Reply:
로그인 로직의 핵심 기능 중 하나인데 Util 클래스를 사용하는것이 만들면서도 찝찝했었는데, 
`JwtToken` 라는 객체를 만들고 이 객체에 위임하는 형식이 더 좋은 것 같습니다!!
감사합니다!